### ISSUE TITLE
Convert from .npy to .ckpt format

### ISSUE THREAD

**Issue by alesolano** [ethereon/caffe-tensorflow#131]
I just created a [repo](https://github.com/alesolano/npy2ckpt) to convert models from .npy to .ckpt (checkpoint) format.

It would be cool if  we could integrate it someway in the caffe-tensorflow repo. The thing is that I needed to change some lines to use Python 3 and TensorFlow 1.

**Comment by seanchung2** [comment]
@alesolano Is your converter only able to deal with googlenet?

**Comment by alesolano** [comment]
No, you can change the parameters of your network (width, height, channels of the input image) and run it.
I have added another example in the repo to make things clearer. It would be easier if you didn't need to deal with these parameters, but I don't know how to avoid it.
